pkg/cloud/services/network/routetables.go

```diff
@@ -214,6 +214,32 @@ func (s *Service) describeVpcRouteTablesBySubnet() (map[string]*ec2.RouteTable,
 	return res, nil
 }
 
+func (s *Service) deleteRouteTable(rt *ec2.RouteTable) error {
```

Comment:
there is nothing new in this function, just isolating from `deleteRouteTables()` to be reused in the error handling for create route.